content/en/docs/usage/gateway-api.md

```diff
@@ -85,38 +101,49 @@ spec:
     kind: Issuer
     group: cert-manager.io
   dnsNames:
-    - example.com
+    - example.com # ✅ Copied from the `hostname` field.
   secretName: example-com-tls
 ```
 
-Contrary to Ingresses, you can set the same secret name multiple times. Let us
-imagine that you have these two listeners:
+## Use cases
+
+### Skipping of invalid listener blocks
+
+Like with an Ingress, the invalid listeners are skipped. For example, no
+Certificate will be created for the following Gateway since both listener that
+does not have a `tls` block won't be used to create the Certificates.
```

Comment:
You are right, I shouln't even talk about "invalid" or "valid", it doesn't really help understanding. The only thing I should say is that you can expect cert-manager to create a Certificate whenever all the above requirements (in the table) are met. Otherwise, cert-manager will just skip this listener block.

Reply:
Done in aa1ecfc18.